src/components/LayerActionIcon/LayerActionIcon.tsx

```
@@ -16,31 +15,11 @@ export function LayerActionIcon({
-          onKeyUp={onKeyUp}
-          className={clsx(style.actionButton, className)}
-          onClick={onClick}
-          tabIndex={0}
-        >
-          {children}
-        </div>
-      </TooltipTrigger>
-      <TooltipContent>{hint}</TooltipContent>
-    </Tooltip>
+    <SimpleTooltip content={hint} placement="top">
+      <button className={clsx(style.actionButton, className)} onClick={onClick}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)